group/assignment5/main.cpp

```diff
@@ -0,0 +1,96 @@
+#include "AlphabetFromDictionary.hpp"
+#include "codeu_test_lib.h"
+
+int main() {
+    vector<string> dictionary;
+    vector<char> alphabet;
+    bool result;
+    unordered_map<char, int> idxs;
+    vector<bool> expectedChars;
+    vector<bool> actualChars;
+
+    // test 1 - assignment example
```

Comment:
Because we don't really know the order of the characters in the alphabet, so we can't compare 2 arrays. But we can use sets for a more efficient test.